### PR TITLE
Feature/avatar

### PR DESCRIPTION
**Pull request by e2o**

### CODE REVIEW

File: src/components/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+import { AvatarIcon, AvatarIconProps } from './AvatarIcon';
+
+export interface AvatarProps extends AvatarIconProps {
+	name?: string;
```
Comment (bertyhell) [inline src/components/Avatar/Avatar.tsx]: missing initials prop

Comment (e2o) [reply src/components/Avatar/Avatar.tsx]: Fixed!

File: src/components/Avatar/Avatar.test.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+import { Avatar } from './Avatar';
+import { AvatarIcon } from './AvatarIcon';
+
+describe('<Avatar />', () => {
+	it('Should be able to render', () => {
+		shallow(<Avatar initials="JD" name="John Doe" />);
+	});
+
+	it('Should correctly render the name', () => {
```
Comment (bertyhell) [inline src/components/Avatar/Avatar.test.tsx]: tests do not show up in the storybook

Comment (e2o) [reply src/components/Avatar/Avatar.test.tsx]: Fixed

**Pull request merged**